src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  display: flex;
+  padding: 0;
+  gap: 57px;
+  font-family: $bodyfont;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+
+  &_imac {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &_name {
+    color: #060b35;
+    font-family: $bodyfont;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 0 16px 4px;
+    line-height: 18px;
+  }
+
+  &_code {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 84px 16px 16px;
+    color: #616070;
+  }
+
+  &_revievs {
```

Comment:
Typo in the class name. It should be '.card__reviews' instead of '.card__revievs'. Make sure class names are spelled correctly to maintain consistency and avoid confusion.